RELEASE.md

```
@@ -0,0 +1,105 @@
+
+```
+<package_name>@<version>
+```
+
+Example:
+```
+xrpl@2.3.1
+```
+
+
+##⚠️ **Important Notes**
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading

(MD018, no-missing-space-atx)